PhysicsTools/PatAlgos/plugins/PATElectronProducer.cc

```diff
@@ -300,29 +312,32 @@ void PATElectronProducer::produce(edm::Event & iEvent, const edm::EventSetup & i
       bool MatchedToAmbiguousGsfTrack=false;
       for (edm::View<reco::GsfElectron>::const_iterator itElectron = electrons->begin(); itElectron != electrons->end(); ++itElectron) {
 	unsigned int idx = itElectron - electrons->begin();
-	if (Matched || MatchedToAmbiguousGsfTrack) continue;
+       auto elePtr = electrons -> ptrAt(idx);
+	     if (Matched || MatchedToAmbiguousGsfTrack) continue;
```

Comment:
looks like indentation changed for no particular reason.
All edits not related to this PR purpose should be removed.
Given the list of not very informative commits in this PR overall, I think that for the final version they will need to be redone into just something finalized so that git history is not polluted.